packages/nimble-components/src/table/specs/README.md

```diff
@@ -265,4 +277,9 @@ Storybook stories will be added to document/showcase the various features and AP
 
 ### Open Issues
 
-None.
+-   How do different invalid configurations on the table get reflected to the client & end user?
+    -   Do all invalid configurations get treated the same way?
+    -   Should we render no rows, show some invalid text, stop updates to the table's state? Some things to keep in mind related to this:
```

Comment:
In the `id-field-name` attribute section of the spec you mention that the table will render no rows when in an invalid state. Could you either delete this open question or clarify that is an initial approach which we plan to revisit when we get further into implementation?

Reply:
I've updated this bullet point to better reflect the open issue.